<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/overlay/denasButtonless.svg);
background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="OASISPro" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>641</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#OASISPro{
border-image: url(:/overlay/oasisPro.svg);
background-color: rgba(255, 255, 255, 0);
border-radius:30px;
}</string>
    </property>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/ok.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/up.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>380</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/down.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/power.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="historyButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>380</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/history.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/buttons/connect.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="_20mins">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/20mins.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="_45mins">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/45mins.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="userDesign">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/userDesign.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="alpha">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/alpha.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="delta">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/delta.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="subDelta">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/subDelta.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="met">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/met.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftCES">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/leftCES.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rightCES">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/rightCES.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #80ff1a;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_1.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #80ff1a;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_2.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #80ff1a;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_3.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: yellow;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_4.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: yellow;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_6.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: red;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_7.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="level8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: red;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_8.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="time1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>390</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.25</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>360</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.50</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0.75</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1.00</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1.50</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="time6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2.00</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="level5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: yellow;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res/oasis.qrc">:/icons/_5.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="powerLight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>340</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background: #afafaf;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>rightCES</zorder>
    <zorder>okButton</zorder>
    <zorder>upButton</zorder>
    <zorder>downButton</zorder>
    <zorder>powerButton</zorder>
    <zorder>historyButton</zorder>
    <zorder>connectButton</zorder>
    <zorder>_20mins</zorder>
    <zorder>_45mins</zorder>
    <zorder>userDesign</zorder>
    <zorder>alpha</zorder>
    <zorder>delta</zorder>
    <zorder>subDelta</zorder>
    <zorder>met</zorder>
    <zorder>leftCES</zorder>
    <zorder>level1</zorder>
    <zorder>level2</zorder>
    <zorder>level3</zorder>
    <zorder>level4</zorder>
    <zorder>level6</zorder>
    <zorder>level7</zorder>
    <zorder>time1</zorder>
    <zorder>time2</zorder>
    <zorder>time3</zorder>
    <zorder>time4</zorder>
    <zorder>time5</zorder>
    <zorder>time6</zorder>
    <zorder>level8</zorder>
    <zorder>level5</zorder>
    <zorder>powerLight</zorder>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #afafaf;</string>
    </property>
    <widget class="QSpinBox" name="leftConnectSpinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="rightConnectSpinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="leftConnectLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QLabel" name="rightConnectLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QLabel" name="BatteryLevelLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Level</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="batteryLevelSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>24.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="leftTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res/oasis.qrc"/>
 </resources>
 <connections/>
</ui>
